src/main/webapp/script.js

```diff
@@ -5,12 +5,20 @@
  * @author tblanshard
  */
 
+let projectID;
+let serviceAccountID;
 
 function initBody() {
     fetch('/data').then(console.log('Works'));
+    projectID = "bt-dataflow-sql-demo";
+    serviceAccountID = "dataflow-service-analyser-sa@bt-dataflow-sql-demo.iam.gserviceaccount.com";
```

Comment:
I think. these should be configuration parameters

Reply:
Good point - I've added a config file and stored those parameters in there instead (#14). Thank you!